pkg/util/tracing/crdbspan.go

```diff
@@ -930,6 +935,9 @@ func (s *crdbSpan) getRecordingNoChildrenLocked(
 					childKey := string(tag.Key)
 					childValue := tag.Value.Emit()
 
+					if rs.Tags == nil {
```

Comment:
Huh, didn't you fix this already?

Reply:
I don't think so, I rebased on master a couple days ago. When we chatted it was me working on this diff its just been sitting in cold storage since. TFTR! I'll wait on Steven to give it a pass since I bugged him to before you stamped it, and then merge away.